inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
```suggestion
def standard_deviation(data):
```
Function name unclear, as well as variable names mmm, devs and s_dev2